src/SFA.DAS.EmployerFinance.Jobs/Program.cs

```diff
@@ -31,7 +31,11 @@ public static async Task Main()
                     .ConfigureWebJobs(b => b.AddAzureStorageCoreServices().AddTimers())
                     .ConfigureAppConfiguration(b => b.AddJsonFile("appsettings.json").AddJsonFile($"appsettings.{environmentName}.json", true).AddEnvironmentVariables())
                     .ConfigureLogging(b => b.AddNLog())
-                    .ConfigureServices(c => c.AddSingleton<IJobActivator>(jobActivator))
+                    .ConfigureServices((context, collection) =>
+                    {
+                        collection.AddSingleton<IJobActivator>(jobActivator);
+                        collection.AddApplicationInsightsTelemetry(context.Configuration);
```

Comment:
Web jobs application insights are configured by calling `AddApplicationInsights` on `ILoggingBuilder`, I think the extension you're using is for ASP.NET.